src/Interactive/EditorFeatures/VisualBasic/Interactive/CompletionProviders/ReplCommandCompletionProvider.vb

```diff
@@ -0,0 +1,66 @@
+' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Threading
+Imports System.Threading.Tasks
+Imports Microsoft.CodeAnalysis.Completion
+Imports Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
+Imports Microsoft.CodeAnalysis.Editor.Implementation.Interactive
+Imports Microsoft.CodeAnalysis.Options
+Imports Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
+Imports Microsoft.CodeAnalysis.Text
+Imports Microsoft.VisualStudio.InteractiveWindow
+Imports Microsoft.VisualStudio.Text.Editor
+Imports Microsoft.VisualStudio.Utilities
+Imports Microsoft.VisualStudio.InteractiveWindow.Commands
+
+Namespace Microsoft.CodeAnalysis.Editor.CSharp.Completion.CompletionProviders
+
+    <ExportCompletionProvider("ReplCommandCompletionProvider", LanguageNames.VisualBasic)>
+    <TextViewRole(PredefinedInteractiveTextViewRoles.InteractiveTextViewRole)>
+    <Order(Before:=PredefinedCompletionProviderNames.Keyword)>
+    Friend Class ReplCommandCompletionProvider
```

Comment:
I'm guessing this is pretty similar to the C# version.  Perhaps they could share a common base type?